tests/testthat/test-apply_theme.R

```diff
@@ -156,15 +156,15 @@ testthat::test_that("T1.18 A warning when `bg` is anything but a character.",{
 
 # Requirement T2 ---------------------------------------------------------------
 
-context("add_theme - T2. The `add_theme` function applies the specified changes to a `ggplot` object.")
+testthat::context("apply_theme - T2. The `apply_theme` function applies the specified changes to a `ggplot` object.")
 
 testthat::test_that("T2.1 No error when a `ggplot` plot is provided, but no theme.",{
   
   gg <- adtte %>%
     visR::estimate_KM("SEX") %>%
     visR::visr()
   
-  testthat::expect_error(visR::add_theme(gg), NA)
+  testthat::expect_error(visR::apply_theme(gg), NA)
```

Comment:
this does not prove that the changes have been applied, only that there are no errors generated

Reply:
you will need to go through ggbuild and see if the theme elements are there

Reply:
True :thinking: Do we already use this non-vdiffr strategy in some other function?

Reply:
I think so but I cant remember. You can do a look in files for ggbuild